esphome/components/electra_rc3_ir/climate.py

```diff
@@ -0,0 +1,14 @@
+import esphome.codegen as cg
+from esphome.components import climate_ir
+
+AUTO_LOAD = ["climate_ir"]
+CODEOWNERS = ["@roygolds"]
+
+electra_rc3_ir_ns = cg.esphome_ns.namespace("electra_rc3_ir")
+ElectraRC3IR = electra_rc3_ir_ns.class_("ElectraRC3IR", climate_ir.ClimateIR)
+
+CONFIG_SCHEMA = climate_ir.climate_ir_with_receiver_schema(ElectraRC3IR)
```

Comment:
You are not using the receive part
```suggestion
CONFIG_SCHEMA = climate_ir.climate_ir_schema(ElectraRC3IR)
```